aws/data_source_aws_organizations.go

```diff
@@ -0,0 +1,65 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/service/organizations"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsOrganizations() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsOrganizationsAccountsRead,
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
```

Comment:
As a Terraform User, I would expect this data source to return Organisation information, not only members.

So first I would expect [describe organisation attributes](https://docs.aws.amazon.com/organizations/latest/APIReference/API_DescribeOrganization.html) and then call the ListAccounts endpoint to get users.
It could even be a separated data source.

Thoughts?